src/Public/Routes.ps1

```diff
@@ -1588,11 +1612,22 @@ function Add-PodeSignalRouteGroup {
         [Parameter()]
         [ValidateSet('Default', 'Error', 'Overwrite', 'Skip')]
         [string]
-        $IfExists = 'Default'
+        $IfExists = 'Default',
+
+        [Parameter(Mandatory = $true, ParameterSetName = 'file')]
```

Comment:
The `[Parameter()]` block for the `$Routes` parameter for `Add-PodeSignalRouteGroup` will need a `ParameterSetName = 'Script'` property being set as well.

And then at the top of the function, the `[CmdletBinding()]` block will need a `DefaultParameterSetName = 'Script'` property being set.

This way, we allow only `Routes` or `FilePath` to be supplied and not both.

Reply:
@Badgerati, why does `Add-PodeStaticRouteGroup` need the parameter? 
Static routes have no script-blocks associated with them

Reply:
`Add-PodeStaticRoute` doesn't, but `Add-PodeStaticRouteGroup` does as it has the `-Routes` scriptblock which is invoked for adding child static routes - similar to `Add-PodeRouteGroup` and `Add-PodeSignalRouteGroup`

Reply:
added